README.md

```diff
@@ -1,45 +1,124 @@
-
-If the commit is only present in `master` (i.e. has no tags), then the commit has yet to be picked up by a release (or it may have been included in a follow up cherry pick for a patch version). You can expect it to be included in the next release candidate that is cut once the designed features have all landed.
-
-## Backporting/cherry-picking of changes to existing builds
-
-From time to time, [backporting](https://en.wikipedia.org/wiki/Backporting) a change to an existing release (including candidates) may be needed. Examples include security issues or critical regressions. If you believe a pull request on `react-native` is a candidate for backporting, please mention it in the version associated _backport request_ issue.
-
-Please note that, if the change hasn't landed on `react-native`'s master, it can't be cherry picked to a release tag yet.
+# React Native’s Release Process
+
+At Facebook React Native is not consumed via released versions, instead they
+always consume the latest version of the `master` branch. This means that they
+are able to iterate fast for _their_ needs; if something is broken the fix is a
+single commit away.
+
+External users are of course similarly able to consume the latest `master`
+version, however this is most likely a too fast moving target for most–aside
+from turn around to get a fix in being longer.
+
+To reconcile these two different use-cases, React Native’s release process is a
+different one from the one many users are familiar with in the JavaScript
+ecosystem.
+
+## The process at a high-level
+
+1. A new version branch is branched off of the `master` branch–hereafter
+   referred to as “the stable branch”.
+1. A [release candidate][RC] is released from the stable branch, allowing users
+   to try it out and provide feedback–hereafter referred to as “the RC”.
+1. Additional changes that have landed in the `master` branch since the stable
+   branch was originally created and are deemed required for the version to be
+   finalized are [cherry-picked][cherry-picking] onto the stable branch. These
+   can be bug-fixes for previously released versions, including the RC, or
+   changes otherwise deemed an improvement for the version to be finalized, such
+   as [UX]/[DX] improvements.
+1. Steps 2 through 3 are repeated until consensus is reached that the version is
+   good enough to release as a stable version.
+1. A stable version is released and communications are sent out to the community
+   informing them of the availability of a new stable version and what changes
+   it includes.
```

Comment:
Maybe in this step I'd be a bit more clear on a couple things:

```
 A stable version is released and communications are sent out to the community
   informing them of its availability - the changes it includes are fully listed in the CHANGELOG.md file. A release entry is created in the GitHub interface and, only for `.0`, a blogpost is published; both highlight the most relevant changes.
```

Reply:
I am definitely touching on those in the detailed doc–which I haven’t pushed yet but this would link there. Do you think that’s good enough? Trying to keep this short and sweet.

Reply:
Yeah I guess it's fine either way - my main point for this comment was to clarify the "two lines" of communication: Changelog.md = in depth / blogpost, releases tab = highlights.

But yeah if you want to keep it short and sweet it's fine.

Reply:
I'm not sure how deep these steps should go but shouldn't the diff generation be mentioned here as well? together with checking that the diff looks _normal_ on upgrade-helper.

Reply:
Not sure if it should be in this bullet point, since it seems we'll have a more in depth doc anyway - and we do a diff for each RC and each stable.

Reply:
It becomes a slippery slope quickly, as shown in this thread :) So for now I’ll leave details about the communications to the other doc, we can see afterwards if that’s good enough.